data/json/overmap/overmap_mutable/homeless_camp_mutable.json

```diff
@@ -7,7 +7,7 @@
     "city_distance": [ 0, 24 ],
     "city_sizes": [ 6, -1 ],
     "occurrences": [ 0, 3 ],
-    "flags": [ "CLASSIC", "URBAN" ],
+    "flags": [ "CLASSIC", "URBAN", "MAN_MADE" ],
```

Comment:
Wouldn't homeless camps be WIlderness instead of urban since there shouldn't be road connections to a homeless encampment in general?

Reply:
Yes, that was an oversight.